app/controllers/welcome_controller.rb

```diff
@@ -0,0 +1,7 @@
+class WelcomeController < ApplicationController
+  def index
+    @movies = Movie.order(votes: :desc).first(10)
```

Comment:
You could DRY this section up by extracting the ordering and selection of top 10 into a model method.